common.tf

```diff
@@ -9,8 +9,9 @@ locals {
   apiary_bucket_prefix             = "${local.instance_alias}-${data.aws_caller_identity.current.account_id}-${data.aws_region.current.name}"
   apiary_assume_role_bucket_prefix = [for assumerole in var.apiary_assume_roles : "${local.instance_alias}-${data.aws_caller_identity.current.account_id}-${lookup(assumerole, "allow_cross_region_access", false) ? "*" : data.aws_region.current.name}"]
   enable_route53_records           = var.apiary_domain_name == "" ? false : true
-
-  datadog_tags = join(" ", formatlist("%s:%s", keys(var.apiary_tags), values(var.apiary_tags)))
+  hms_readwrite_namespace          = var.hms_readwrite_namespace
```

Comment:
isn't it better to call this hms_rw_dd_namespace and hms_ro_dd_namespace? Because since we run in kubernetes, anything related to namespace points to a kubernetes namespace. Perhaps it even makes sense to have a common variable hms_dd_namespace since most of the time we would need to store the metrics with the same prefix (i.e. namespace).